docker/hive2-kerberos/Dockerfile

```diff
@@ -0,0 +1,92 @@
+FROM keboola-hive2-no-auth
```

Comment:
Dockerfiles a configs som dal dokopy z internetu, .... vela veci bolo starych a nefunkcnych a musel som to rozne ladit.
Myslim si, ze to staci rychlo prebehnut. Zaklad je, ze sa daju oproti tomu spustat PHP testy, tak ako doteraz.